doc/user/layouts/shortcodes/beta.html

```diff
@@ -4,4 +4,10 @@
     is in beta. It may have performance or stability issues and is not subject
     to our <a href="{{ "/versions/#backwards-compatibility" | relURL }}">backwards compatibility</a>
     guarantee.
+
+    {{if in (apply $.Site.Params.versions "index" "." "name") (.Get 0)}}
```

Comment:
@benesch Was is a conscious choice to not have the version in the beta shortcode, as we have in experimental?